lib/options.js

```diff
@@ -163,6 +163,15 @@ module.exports = {
         description: 'Whether to show mismatches for incorrect name and description of request',
         external: true,
         usage: ['VALIDATION']
+      },
+      {
+        name: 'Ignore mismatch for unresolved postman variables',
+        id: 'ignoreUnresolvedVariables',
+        type: 'boolean',
+        default: false,
+        description: 'Whether to ignore mismatch thrown upon validation of unresolved collection/environment variable',
```

Comment:
`Whether to ignore mismatches resulting from unresolved variables in the Postman request`